Steepfile

```diff
@@ -0,0 +1,8 @@
+target :lib do
+  signature "sig"
+
+  check "lib"
+
+  # ignore whole of lib, to just validate RBS in sig
+  ignore "lib"
```

Comment:
It works simply omitting the `check` entry.
```suggestion
```
Also, consider pessimistically enable all diagnostics.
```suggestion
  configure_code_diagnostics Steep::Diagnostic::Ruby.all_error
```